Client/src/Pages/PageSpeed/CreatePageSpeed/index.jsx

```diff
@@ -39,6 +40,7 @@ const CreatePageSpeed = () => {
   });
   const [https, setHttps] = useState(true);
   const [errors, setErrors] = useState({});
+  const [isCheckingEndpoint, setIsCheckingEndpoint] = useState(false);
```

Comment:
Since you've added an async thunk to the redux store we may as well use the redux loading state rather than handling this manually.

```
	const { monitors, isLoading } = useSelector((state) => state.uptimeMonitors);
```

that will be true whenever any of the async operations is happening